product_assortment/models/ir_filters.py

```diff
@@ -39,6 +43,25 @@ def _get_default_is_assortment(self):
     record_count = fields.Integer(compute="_compute_record_count")
 
     is_assortment = fields.Boolean(default=lambda x: x._get_default_is_assortment())
+    partner_domain = fields.Text(default="[]", required=True)
+    all_partner_ids = fields.Many2many(
```

Comment:
Currently this is never seen by the user. Is it intended? Or could another button be added like this?
![image](https://user-images.githubusercontent.com/38977934/124775087-3d068e80-df36-11eb-9ab2-acf3a1f5f80f.png)

Reply:
Yes @joao-p-marques This is only a technical field to be use in other modules like as sale_order_product_assortment
Maybe we could add the smartbutton to open related partners

Reply:
The domain widget shows a button to access the records, so to make it homogeneous, perhaps the current smartbutton can be removed.